### PR TITLE
Nerfs TEG by a flat factor of 3 power output

### PR DESCRIPTION
**Pull request by JamieD1**
TEG Is way too easy to get ridiculous energy amounts without any brain power.

This is still close to 700KW without modifying any pipes roundstart which is enough considering Supermatter is about 350 KW

:cl:  
tweak: TEG now Gives 1/3 Energy
/:cl:

### CODE REVIEW

**Comment by Simplehorror** [comment]
so instead of using the efficency variable (2 lines above) you are just doing that

File: code/modules/power/generator.dm [hunk 1]
```diff
@@ -91,7 +91,7 @@
 				var/energy_transfer = (hot_subsection_temp - cold_subsection_temp)*internal_heat_cap
 
 				//produce electricity
-				lastgen += energy_transfer*efficiency
+				lastgen += (energy_transfer*efficiency) / 3
```
Comment (ToasterBiome) [inline code/modules/power/generator.dm]:
```suggestion
				lastgen += (energy_transfer*efficiency)
```

Comment (ToasterBiome) [reply code/modules/power/generator.dm]: just change efficiency from 0.65 to .2-.22

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by Simplehorror** [comment]
why are we doing this when people can barely set it up

**Comment by JamieD1** [comment]
> so instead of using the efficency variable (2 lines above) you are just doing that

Because that changes heat as well, this is just to do with power.

File: code/modules/power/generator.dm (hunk 1 above)
Comment (JamieD1) [reply code/modules/power/generator.dm]: > just change efficiency from 0.65 to .2-.22


Because that changes heat as well, this is just to do with power.

**Comment by Cartlord** [comment]
Why'd you selfmerge this when getting the T.E.G. working is already a pain? And why does it matter how much power it makes, when any quantity of power over that which the station needs is effectively useless?

**Pull request merged**